Dockerfile.std

```diff
@@ -0,0 +1,23 @@
+# Generated by: tyk-ci/wf-gen
+# Generated on: Friday 19 February 2021 01:16:15 PM UTC
+
+# Generation commands:
+# ./pr.zsh -branch goreleaser/more -base goreleaser/more -title Address review #2 -p
+# m4 -E -DxREPO=tyk-pump
+
+FROM debian:buster-20210208-slim
+RUN apt-get update \
+ && apt-get dist-upgrade -y ca-certificates \
+ && apt-get autoremove -y
+COPY *.deb /
+RUN dpkg -i /*.deb && rm /*.deb
+
+ARG PORTS
+EXPOSE $PORTS
+VOLUME [ "/opt/tyk-pump" ]
```

Comment:
Same about the volume exposure as in the other PRs of this set.

Reply:
Done.